windows/hid.c

```diff
@@ -109,10 +109,146 @@ static struct hid_api_version api_version = {
 		USHORT Reserved[17];
 		USHORT fields_not_used_by_hidapi[10];
 	} HIDP_CAPS, *PHIDP_CAPS;
-	typedef void* PHIDP_PREPARSED_DATA;
+	typedef enum _HIDP_REPORT_TYPE {
+		HidP_Input,
+		HidP_Output,
+		HidP_Feature,
+		NUM_OF_HIDP_REPORT_TYPES
+	} HIDP_REPORT_TYPE;
+	/// <summary>
+	/// Contains information about one global item that the HID parser did not recognize.
+	/// </summary>
```

Comment:
I'm not even familiar with this documentary style.
HIDAPI alreayd uses doxygen-style syntax for documentary comments, please use it as well.

Reply:
Ok, after a brief googling I understand, that in `hidapi.h` we actually have more like a javadoc-style syntax (which _is_ supported by doxygen), but my point is - please be consistent.